### ISSUE TITLE
Release to firefox add-ons ?

### ISSUE THREAD

**Issue by abcfy2** [EliverLara/firefox-nordic-theme#8]
Your theme is really amazing. Would you please release to firefox add-ons ( https://addons.mozilla.org/firefox/ )

Thanks.

**Comment by aphive** [comment]
Agreed here..  Using the chrome directory or running the install script method doesn't always work, mine borked my setup to the point I had to do a Refresh to get things back.

Having this as an actual add-on would be great.
